netci.groovy

```diff
@@ -2861,6 +2861,7 @@ Constants.allScenarios.each { scenario ->
                                 // HACK -- Arm64 does not have corefx jobs yet.
                                 // Clone corefx and build the native packages overwriting the x64 packages.
                                 if (architecture == 'arm64') {
+                                    shell("mkdir -p ./bin/CoreFxNative")
```

Comment:
Direct copy from below
https://github.com/dotnet/coreclr/blob/235a34673994560deb6011be08aff18b4399658b/netci.groovy#L2867-L2868

Reply:
You got the path wrong: it should be "./bin/CoreFxBinDir" to match the copy below.

Reply:
Thanks.  I assumed I wrote the code and would have chosen the same name...  Blind....

Reply:
Done